nginx_stage/share/nginx_stage_example.yml

```diff
@@ -52,7 +52,7 @@
 
 # Path to system-installed NGINX binary
```

Comment:
Since we're doing this to potentially allow parallel installations the comments about 'system installed' NGINX's should be replaced with something like:

```yaml
# Path to NGINX executable used by OnDemand
```

Reply:
Updated a few places that used `system-installed`.